### PR TITLE
chore: Bump ajv to ^8.1.0

### PR DESCRIPTION
**Pull request by ukstv**

### CODE REVIEW

File: packages/core/src/utils.ts [hunk 1]
```diff
@@ -1,4 +1,6 @@
-import ajv from "ajv"
+import Ajv from "ajv"
+import addFormats from "ajv-formats"
```
Comment (oed) [inline packages/core/src/utils.ts:2]: What does this function do?

**Review by oed — COMMENTED** [review]
Just a question.

File: packages/core/src/utils.ts (hunk 1 above)
Comment (ukstv) [reply packages/core/src/utils.ts:2]: It adds "custom" formats defined in later versions of JSON schema. For example, "date-time". Earlier it was part of the core `ajv` package.

Comment (ukstv) [reply packages/core/src/utils.ts:2]: > From version 7 Ajv does not include formats defined by JSON Schema specification - these and several other formats are provided by ajv-formats (opens new window) plugin.

https://ajv.js.org/guide/formats.html

Comment (oed) [reply packages/core/src/utils.ts:2]: ok

**Review by oed — APPROVED** [review]

**Pull request merged**